2018/07.md

```diff
@@ -91,6 +91,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
 
     | stage | timebox | topic | presenter |
     |:-----:|:-------:|-------|-----------|
+    | 2     | 45m     | [`Intl.NumberFormat` Unified Feature Proposal](https://github.com/sffc/proposal-unified-intl-numberformat) | Shane Carr |
```

Comment:
The current stage is 0; you’ll want to update this 2 to 0, and indicate in the topic that it’s “for stage 2”

Reply:
(i went ahead and rebased+fixed this for you)